### DISCUSSION TITLE
How to merge branches in git?

### DISCUSSION THREAD

**Discussion by anna-proactive** [svmed2050/ballons-test#1] [Q&A] [answered]
I have a master branch and a working branch_1. I want to 'move' branch_1 exactly as it is to master. So I want something like this:

git checkout master
git merge branch_1 # I don't know what is correct...

Can you help me?

**Comment by svmed2050** [comment, marked as answer]
Conflicts are going to happen if both branches have changes to the files. This is a good thing. Keeping your branches up-to-date with each other will prevent some of them . However over all, conflicts are not bad. The rebase option can also prevent many of them from happening.

git merge branch_1
If you are on master, merging will bring the changes as you expect.

You could also

git rebase branch_1
This will take the changes from branch_1 and append them to master without a merge commit.
